tests/filters/test_addslashes.py

```diff
@@ -29,10 +28,10 @@ def test_addslashes01(self):
 @pytest.mark.xfail(reason="autoescape not there yet")
 def test_addslashes02(self):
     """@setup({"addslashes02": "{{ a|addslashes }} {{ b|addslashes }}"})"""
-    template = "{{ a|addslashes }} {{ b|addslashes }}"
+    # template = "{{ a|addslashes }} {{ b|addslashes }}"
 
-    django_template = engines["django"].from_string(template)
-    rust_template = engines["rusty"].from_string(template)
+    # django_template = engines["django"].from_string(template)
+    # rust_template = engines["rusty"].from_string(template)
```

Comment:
Commenting this out defeats the point of `xfail` in my opinion.

Reply:
I changed to using `# noqa` instead. Either way the test is failing.